### DISCUSSION TITLE
Question on payment method (PayNow)

### DISCUSSION THREAD

**Discussion by Sam9913** [flutter-stripe/flutter_stripe#1115] [Q&A] [answered]
Thanks author for making my life easier when implementing Stripe with flutter! Very appreciate that, but I'm just wondering is it possible for me to use PayNow with this?

Just for your information that I have try PaymentMethodParams.fromJson(/*json here*/); but I don't really understand how it works. Any help will be appreciate.

**Comment by remonh87** [comment, marked as answer]
Hi,

As far as I can see the paynow does not require a native sdk implementation. I also would double chech this with stripe

**Reply by tran-huy-phuc** [reply]
Hi @Sam9913 did you find a solution?

@remonh87 what do you mean by "not require a native sdk implementation"? Did you get any chance to check with Stripe?

**Reply by Sam9913** [reply]
> Hi @Sam9913 did you find a solution?
> 
> @remonh87 what do you mean by "not require a native sdk implementation"? Did you get any chance to check with Stripe?

Hi, I forgot the details but I remember the solution that I did is integrate paynow at the backend.

Since they don't really need any integration on mobile (according to stripe website documentation).

**Reply by tran-huy-phuc** [reply]
Thank you @Sam9913 

Integrating it on the backend sounds reasonable. On the documentation, I also see the code provided with some "backend" languages (Ruby, Python, PHP, Java...)

**Reply by tran-huy-phuc** [reply]
Hi @Sam9913  PayNow is not supported on native SDK (Android): https://github.com/stripe/stripe-android/issues/6840#issuecomment-1792433365

How did you implement it for Android?

**Reply by Sam9913** [reply]
> Hi @Sam9913  PayNow is not supported on native SDK (Android): https://github.com/stripe/stripe-android/issues/6840#issuecomment-1792433365
> 
> How did you implement it for Android? 

Good question, I haven't using stripe for around a year. I have no idea how they continue with that 😅 Sorry that I can't help on that
